nexttrace.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+installdir=`flatpak-spawn --host flatpak info --show-location io.github.Archeb.opentrace`
```

Comment:
this script looks very wrong

Reply:
I know that's pretty dirty but what else can we do, except for not packaging it as a flatpak ...